source/jormungandr/jormungandr/exceptions.py

```diff
@@ -56,11 +56,11 @@ def __init__(self, region=None, lon=None, lat=None, object_id=None, custom_msg=N
             elif object_id[:6] == "coord:":
                 lon, lat = object_id[6:].split(":")
                 object_id = None
-        if region == lon == lat is None == object_id:
+        if not any([region, lon, lat, object_id]):
```

Comment:
`not any([...])` should be expressed as `none_of([...])` really, it is much easier to read and understand.
 
It's weird that Python hasn't such a built-in function :( Maybe this exists in another lib ?

Reply:
I guess that there's no point in having two opposite functions when there's `not`